docusaurus/docs/develop/path/7_e2e_tests.md

```diff
@@ -1,94 +1,187 @@
-- Services respond with expected data within acceptable latency thresholds
-- The system maintains high availability and reliability under load
-- Response metrics meet predefined success criteria
+E2E tests need a valid config file in `./e2e`:
 
-## Running Tests
+- `./e2e/morse.config.yaml` for Morse
+- `./e2e/shannon.config.yaml` for Shannon
 
-:::important E2E 
+Config must match the protocol/services you want to test.
 
-As the E2E tests send actual relay requests, they require a valid configuration file that:
-- is configured for the protocol being tested
-- contains valid configuration for the services being tested
+See [PATH Configuration File docs](./5_configurations_path.md) for details.
 
-These files must be located in the `./e2e` directory and be named after the protocol:
-- `.morse.config.yaml` for Morse
-- `.shannon.config.yaml` for Shannon
+:::warning TODO: Prebuilt configs
 
-For instructions on how to create these configuration files, see the [PATH Configuration File documentation](./5_configurations_path.md).
+TODO(@olshansk): Provide prebuilt configs for 1-minute onboarding.
+
+:::
 
-:::tip
+### Helpers to create configurations
 
-Make targets are available for each protocol to copy example configuration files to the `./e2e` directory:
+You can use the following commands to copy example configs and follow the instructions in your CLI:
 
 - `make morse_prepare_e2e_config`
 - `make shannon_prepare_e2e_config`
 
-Once copied, you will need to update the configuration with valid values before running the tests.
+<details>
+<summary>🌿 For Grove Employees</summary>
 
-:::
+Search for `E2E Config` in `1Password` and copy-paste those configs directly.
 
-### Make Targets
+</details>
 
-Tests can be executed using Make targets defined in the project:
+## Helper Make Targets
 
-```bash
-# Run all tests (unit and E2E)
-make test_all
+:::tip make help
+
+Run `make help` to see all available make targets.
+
+:::
 
-# Run only E2E tests for Shannon
+**Only E2E tests for Shannon:**
+
+```bash
 make test_e2e_evm_shannon
+```
 
-# Run only E2E tests for Morse
+**Only E2E tests for Morse:**
+
+```bash
 make test_e2e_evm_morse
+```
+
+**E2E for a specific service** (if you know which service ID you want to test):
 
-# Run E2E tests for a specific service
+```bash
 make test_e2e_evm_morse SERVICE_ID_OVERRIDE=F021
+```
+
+**E2E against local PATH binary** (no Docker):
 
-# Run E2E tests against a PATH binary running locally without Docker
+```bash
+# Run make path_run in another shell
 make test_e2e_evm_morse GATEWAY_URL_OVERRIDE=http://localhost:3069/v1
+```
+
+**Force Docker rebuild** (if you made changes to the code):
 
-# Force Docker rebuild
+```bash
 make test_e2e_evm_morse DOCKER_FORCE_REBUILD=true
+```
 
-# Enable Docker logs
-make test_e2e_evm_morse DOCKER_LOG=true
+\*_Disable Docker logs_ (if its slowing down your system):
 
-# Wait 30 seconds for hydrator checks
+````bash
+make test_e2e_evm_morse DOCKER_LOG=false
```

Comment:
Ditto; this doesn't work to override. IMO DOCKER_LOG should not be enabled by default.